brave/browser-laptop
# Adds warning dialog when you have active downloads

Submitter Checklist:


Resolves #2707

Auditors:

Test Plan:
- covered with automated tests

Manual Plan:
- download a big file (ubuntu for example)
- try to quite brave, you should see the warning dialog


Reviewer Checklist:

Tests




Dialog that we show now:
![image](https://user-images.githubusercontent.com/9574457/29769354-46d995c2-8bea-11e7-817c-fcd4f50f827a.png)
